testify/test_discovery.py

```diff
@@ -31,18 +30,48 @@ class DiscoveryError(TestifyError):
     pass
 
 
-def gather_test_paths(testing_dir):
-    """Given a directory path, yield up paths for all py files inside of it"""
-    for adir, subdirs, subfiles in os.walk(testing_dir):
-        # ignore .svn directories and other such hiddens
-        if adir.startswith('.'):
+def to_module(path):
+    path = os.path.relpath(path)
+
+    if path.startswith('..'):
+        raise DiscoveryError('Test outside of current directory')
```

Comment:
I don't see any particular reason for this case to be an error, but as long as it's not a regression relative to the parent changeset, it doesn't really matter.